src/test/java/com/microsoft/sqlserver/testframework/Constants.java

```diff
@@ -25,6 +25,7 @@ private Constants() {}
      * NTLM  - - - - - - - For NTLM tests
      * reqExternalSetup  - For tests requiring external setup
      * clientCertAuth  - - For tests requiring client certificate authentication setup
+     * Fedauth - - - - - - For Fedauth tests

```

Comment:
I think we shouldn't capitalize Fedauth to maintain consistency of the test tags